pkg/sink/sink.go

```diff
@@ -68,20 +68,21 @@ func (r Resource) HandleEvent(response http.ResponseWriter, request *http.Reques
 		return
 	}
 
-	log.Printf("EventListener: %s in Namespace: %s handling event with payload: %s and header: %v",
-		r.EventListenerName, r.EventListenerNamespace, string(event), request.Header)
+	eventId := template.Uid()
+	log.Printf("EventListener: %s in Namespace: %s handling event (EventID: %s) with payload: %s and header: %v",
+		r.EventListenerName, r.EventListenerNamespace, eventId, string(event), request.Header)
 	// Execute each Trigger
 	for _, trigger := range el.Spec.Triggers {
-		go r.executeTrigger(event, request.Header, trigger)
+		go r.executeTrigger(event, request.Header, trigger, eventId)
 	}
 	fmt.Fprintf(response, "EventListener: %s in Namespace: %s handling event with payload: %s and header: %v",
```

Comment:
Can you add Event ID here also and remove the header and payload? I was thinking of removing the header. I realise that they are not a useful field to receive.

Reply:
I make changes.